x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/upgrade_prebuilt_rules_table/translations.ts

```diff
@@ -24,6 +24,27 @@ export const UPDATE_SELECTED_RULES = (numberOfSelectedRules: number) => {
   );
 };
 
+export const BULK_UPDATE_BUTTON_TOOLTIP_NO_PERMISSIONS = i18n.translate(
+  'xpack.securitySolution.detectionEngine.rules.upgradeRules.bulkButtons.noPermissions',
+  {
+    defaultMessage: "You don't have permissions to update rules",
+  }
+);
+
+export const BULK_UPDATE_ALL_RULES_BUTTON_TOOLTIP_CONFLICTS = i18n.translate(
+  'xpack.securitySolution.detectionEngine.rules.upgradeRules.bulkButtons.allRules.conflicts',
+  {
+    defaultMessage: 'All rules have conflicts. Please update them individually.',
```

Comment:
In the second sentence, are we advising users to fix the field conflicts individually? Or should they update the rules individually and fix the conflicts during the update process? (Sorry for all the questions. I'm still learning about this feature and its associated user flows.)

The following edit removes "please" because we don't normally use it in copy. (Refer to the guidelines in the "Avoid using 'please'" section [here](https://eui.elastic.co/#/guidelines/writing/guidelines#messages).)

```suggestion
    defaultMessage: 'All rules have conflicts. Update them individually.',
```